File: 2.0/java/texts/getTextBroadcast.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.TextBroadcast;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        TextBroadcast broadcast = client.textBroadcastsApi().get(123L);
    }
}

File: 2.0/csharp/texts/getTextBroadcast.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        TextBroadcast broadcast = client.TextBroadcastsApi.Get(379506003, "name,status,labels");
    }
}
